test/text.js

```diff
@@ -407,6 +417,16 @@ describe('bodyParser.text()', function () {
       test.expect(200, '"name is 论"', done)
     })
 
+    if (hasBrotliSupport) {
```

Comment:
Instead of excluding the test, have the test always be there but show up as skipped in the mocha output when the support is missing.